packages/flutter_test/test/widget_tester_test.dart

```diff
@@ -178,215 +178,6 @@ void main() {
       expect(logPaints, <int>[60000, 70000, 80000]);
     });
   });
-
```

Comment:
To review the changes to these tests, [this commit](https://github.com/flutter/flutter/pull/127137/commits/7b63da59cb33741404de4efe601bcd630c2b0a0e) has _just_ the test updates in it.